{% extends "email/base.html" %}
{% load email_utils %}
{% load data_capture %}

{% block content %}
<h1>
  There's a problem with your upload
</h1>
<p>
  Unfortunately we found an error when processing the Region 10 data you uploaded on {{ upload_source.created_at|tz_timestamp }}. Please visit CALC to <a href="{{ r10_upload_link }}">try again</a>.
</p>
<p>
  Also, this "traceback" may help the development team debug the problem:
  <pre>
    {{ traceback }}
  </pre>
</p>
  {% cta url=r10_upload_link action_text="Try uploading again" %}
{% endblock %}
